docs-gb/installation/production-environment/ingress-controller/istio.md

```diff
@@ -0,0 +1,160 @@
+
+    ```
+     apiVersion: networking.istio.io/v1alpha3
+     kind: Gateway
+     metadata:
+       name: my-gateway
+       namespace: seldon-mesh
+     spec:
+       selector:
+         istio: ingressgateway # Use Istio's default ingress gateway
+       servers:
+       - port:
+           number: 80
+           name: http
+           protocol: HTTP
+         hosts:
+         - "*"
+    ```
+3.  Change to the directory that contains `istio-seldon-gateway.yaml` file and apply the configuration:
+
+    ```
+    kubectl apply -f istio-seldon-gateway.yaml
+    ```
+
+    When the configuration is applied, you should see this:
+
+    ```
+    gateway.networking.istio.io/seldon-gateway created
+    ```
+4.  Find the IP address of the Seldon Core 2 instance running with Istio:
+
+    ```
+    ISTIO_INGRESS=$(kubectl get svc seldon-mesh -n seldon-mesh -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+    
+    echo "Seldon Core 2: http://$ISTIO_INGRESS"
+
+    ```
+
+    {% hint style="info" %}
+    Make a note of the IP address that is displayed in the output. This is the IP address that you require to try out the [Kubernetes Examples](/docs-gb/examples/k8s-examples.md).
+    {% endhint %}
+
+### Expose Seldon mesh service
+
+It is important to expose `seldon-service` service to enable communication between deployed machine learning models and external clients or services. The Seldon Core 2 inference API is exposed through the `seldon-mesh` service in the `seldon-mesh` namespace. If you install Core 2 in multiple namespaces, you need to expose the `seldon-mesh` service in each of namespace.
+
+1.  Verify if the `seldon-mesh` service is running for example, in the namespace `seldon`.
+
+    ```bash
+    kubectl get svc -n seldon-mesh
+    ```
+
+    When the services are running you should see something similar to this:
+
+    ```bash
+    mlserver-0               ClusterIP      None             <none>          9000/TCP,9500/TCP,9005/TCP                                                                  43m
+    seldon-mesh              LoadBalancer   34.118.225.130   34.90.213.15    80:32228/TCP,9003:31265/TCP                                                                 45m
+    seldon-pipelinegateway   ClusterIP      None             <none>          9010/TCP,9011/TCP                                                                           45m
+    seldon-scheduler         LoadBalancer   34.118.225.138   35.204.34.162   9002:32099/TCP,9004:32100/TCP,9044:30342/TCP,9005:30473/TCP,9055:32732/TCP,9008:32716/TCP   45m
+    triton-0                 ClusterIP      None             <none>          9000/TCP,9500/TCP,9005/TCP 
+    ```
+2.  Create a YAML file to create a VirtualService named `iris-route` in the namespace `seldon-mesh`. For example, create the `seldon-mesh-vs.yaml` file. Use your preferred text editor to create and save the file with the following content:
+
+    ```yaml
```

Comment:
I believe there are a few issues with this manifest. I got the error `error: error parsing seldon-mesh-vs.yaml: error converting YAML to JSON: yaml: line 14: did not find expected '-' indicator`.

* Unused headers field
* Indentations seems to be off
* **host** indentation under destination

Reply:
Hi @paulb-seldon  just committed the changes. Please confirm if it works.

Reply:
It seems that it works but should the host be `seldon-mesh.seldon-mesh.svc.cluster.local` or just `seldon-mesh.svc.cluster.local` ?

Reply:
Hi @paulb-seldon  good question. May be Chandra can help us with the answer.

Reply:
Nevermind, it looks like `seldon-mesh.seldon-mesh.svc.cluster.local` is correct, my mistake! (https://docs.seldon.io/projects/seldon-core/en/v2.8.3/contents/kubernetes/service-meshes/istio/index.html)